gcloud-java-storage/src/test/java/com/google/gcloud/storage/ITStorageTest.java

```diff
@@ -86,7 +86,8 @@ public static void afterClass() throws ExecutionException, InterruptedException
 
   @Test(timeout = 5000)
   public void testListBuckets() throws InterruptedException {
-    Iterator<BucketInfo> bucketIterator = storage.list(Storage.BucketListOption.prefix(BUCKET),
+    Iterator<Bucket> bucketIterator =
```

Comment:
Not related to this line.

In this file we could substitute `Blob` to `BlobInfo` (and `Bucket` to `BucketInfo`) also to hold the result of `create` and `get` methods. That way we can use `blob.delete()` rather than `storage.delete(bucketName, blobName)`